docs/release-notes/software-releases.md

```diff
@@ -18,6 +18,23 @@ None to report.
 
 None to report.
 ------>
+## April 27, 2021: Clinical Submission and Program Dashboard Updates
+Data Platform v1.93.0
+API v 3.16.0
+
+### New Features 
+- A New version of the [Program Dashboard](/docs/submission/submitted-data) has been released to help report on data submission and molecular data processing progress has been released. Updates to this section includes: 
+    - Redone program statistics bar
+	- Donor clinical completeness tracking chart showing a programs progress over times towards submitting complete clinical data for all committed program donors. 
+	- Donor molecular processing completeness chart showing processing progress over times towards having completed molecular processing  complete for all committed program donors. 
```

Comment:
suggested rewording: `Donor molecular summary chart showing molecular processing progress over times, working towards having completeness for all committed program donors`.